packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -547,14 +547,8 @@ export default function ScheduleDetails({ modalProps, actions, id }) {
 
           {state.upcomingDates && (
             <View style={{ fontSize: 13, marginTop: 20 }}>
-              <Text style={{ color: colors.n4, fontWeight: 600 }}>
-                Upcoming dates
-              </Text>
-              <Stack
-                direction="column"
-                spacing={1}
-                style={{ marginTop: 10, color: colors.n4 }}
-              >
+              <Text style={{ fontWeight: 600 }}>Upcoming dates</Text>
+              <Stack direction="column" spacing={1} style={{ marginTop: 10 }}>
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong colors

before:
<img width="164" alt="Screenshot 2023-08-12 at 09 30 21" src="https://github.com/actualbudget/actual/assets/886567/23d00984-cd0b-499a-9552-556470ae27a1">


after:
<img width="168" alt="Screenshot 2023-08-12 at 09 30 19" src="https://github.com/actualbudget/actual/assets/886567/0b6ff818-8dd8-4261-8f2a-2b89336b7339">